double_element_in_array2.rb

```diff
@@ -0,0 +1,9 @@
+#Use the map function to double all the elements in the array.
+# However, handle edge cases (like array can have a character) as well.
+array = [1 , 2 , 3 , "a" , "b"]
+new_array = array.map { |n|
+if n.kind_of? Integer
+  n * 2
+end
+ }
```

Comment:
use do..end syntax and single line syntax for conditional statement

Reply:
Done